documentation/API proposals/APIproposal_QualityByDesign_CableLabs.md

```diff
@@ -0,0 +1,16 @@
+| API family owner                          | CableLabs                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| API summary                               | API allowing 3rd party applications to:<br>- report appliction KPIs to network operator<br>- receive QbD score<br>- receive root cause and recommended corrective actions<br>- request service improvements                                                                                                                                                                                                                                                                                                                                                                                         |
+|                                           | **API Scope**<br>This API would be part of the Home Device API Family. The QbD API allows a 3rd party application developer to create a session with a network operator. The application can then share KPIs such as:<br>- latency<br>- jitter<br>- packet loss<br>- bitrate<br>The network operator can monitors these KPIs and determine a QbD score. If the score ever drop below certain thresholds, an event can be triggered for rapid telemetry collection. The network operator can then determine root cause and suggest corrective actions either to the 3rd party app or to the network. |
+|                                           | **Use Case Example 1 - Wi-Fi Congestion**<br>Wi-Fi congestion at a `Service Site` is affecting video conferencing performance, reflecting lower QbD scores and poor video quality                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
I suspect that many of the MNO's will think of a service site a cell tower.  Could we provide a definition or more common phrase.

Maybe just link to the NetworkAccessManagement definition.  https://github.com/camaraproject/NetworkAccessManagement/blob/1c804a5159f67370f005dbf0cc82e5c7e72725f0/code/API_definitions/network_access_management.yaml#L40

Reply:
ok, linked to that definition to clarify